### ISSUE TITLE
Remove pyc files from source distribution on PyPI

### ISSUE THREAD

**Issue by moubctez** [twolfson/restructuredtext-lint#56]
I found these in restructuredtext_lint-1.3.0.tar.gz downloaded from PyPI:
```
restructuredtext_lint-1.3.0/restructuredtext_lint/test/__init__.pyc
restructuredtext_lint-1.3.0/restructuredtext_lint/test/test.pyc
restructuredtext_lint-1.3.0/restructuredtext_lint/cli.pyc
restructuredtext_lint-1.3.0/restructuredtext_lint/__init__.pyc
restructuredtext_lint-1.3.0/restructuredtext_lint/lint.pyc
```
Please, get rid of these files. Thanks :-)

**Comment by twolfson** [comment]
Huh, I figured bundlers would automatically ignore those ._. Looking into scrubbing them :+1:

**Comment by twolfson** [comment]
This has been patched in `1.3.1`. Thanks for the bug report =)
